fili-core/src/main/java/com/yahoo/bard/webservice/data/HttpResponseChannel.java

```diff
@@ -58,16 +35,15 @@ public HttpResponseChannel(
      * @param apiRequest  Api request object with all the associated info with it
      * @param httpResponseMaker  Helper class instance to prepare the response object
      *
-     * @deprecated  The ResponseFormatType and UriInfo should be passed explicitly, rather than implicitly via the
```

Comment:
And it actually doesn't it make that much harder to setup a test since Groovy ignores access modifiers and the `DataApiRequest` (at least) has a private default constructor. So you can build an empty `ApiRequest` and then use the request's withers to initialize only the pieces you care about for the test.